### PR TITLE
Set adblock agents

### PR DESCRIPTION
**Pull request by scripthunter7**

### CODE REVIEW

**Review by piquark6046 — COMMENTED** [review]
Is it necessary to apply additional adblock syntax support of GitHub?

**Comment by scripthunter7** [comment]
@piquark6046 In short, no. The .gitattributes file is enough, **but** Linguist has popularity requirements (and popularity can be revised at any time), so it would be good if as many filters as possible started with adblock agents. Currently, the adblock language [based on agents](https://github.com/github/linguist/blob/bf853f1c663903e3ee35935189760191f1c45e1c/lib/linguist/heuristics.yml#L681-L700), so I recommend `.gitattributes` only in justified cases. So if that's not a problem for you, please merge the PR.

File: filter_Extension.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[uBlock Origin]
```
Comment (piquark6046) [inline filter_Extension.txt]: Please change to `[AdGuard; uBlock Origin]`

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filter_Extension.txt (hunk 1 above)
Comment (piquark6046) [reply filter_Extension.txt]:
```suggestion
[AdGuard; uBlock Origin]
```

Comment (scripthunter7) [reply filter_Extension.txt]: Done

Comment (scripthunter7) [reply filter_Extension.txt]: Sorry, I saw the suggestion later

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**